### DISCUSSION TITLE
Markup errors as errors and warnings as warnings

### DISCUSSION THREAD

**Discussion by HiranChaudhuri** [4ian/GDevelop#3531] [Enhancement ideas (and their technical discussions)]
During my project build I am getting messages like

```
copyFile(/project/images/strip_grass.png, /output/strip_grass.png) failed: Error: ENOENT: no such file or directory, stat '/project/images/strip_grass.png'
copyFile(/project/images/strip_water.png, /output/strip_water.png) failed: Error: ENOENT: no such file or directory, stat '/project/images/strip_water.png'
copyFile(/project/sounds/plane_explodes.wav, /output/plane_explodes.wav) failed: Error: ENOENT: no such file or directory, stat '/project/sounds/plane_explodes.wav'
```

According to https://github.com/arthuro555/gdexporter/issues/7 they are just warnings, which means the fact that GDevelop finishes building the project is ok.

Would it be possible to change the word "error" to "warning" in such cases?

**Comment by 4ian** [comment]
Yes there is a real need to improve the error reporting so that errors/warnings during exports are exposed in the interface. There is a start of a discussion here: https://github.com/4ian/GDevelop/discussions/2919#discussioncomment-1185361
And a card on the roadmap: https://trello.com/c/IiLgNR16/462-add-a-diagnostic-report-to-warn-about-potential-issues-in-the-game-and-show-them-in-the-events-sheet
